jwst/pipeline/calwebb_spec2.cfg

```diff
@@ -9,6 +9,8 @@ save_results = True
       [[imprint_subtract]]
       [[msa_flagging]]
       [[extract_2d]]
+      [[master_background]]
```

Comment:
The master_background step itself never actually gets called, even when processing MOS, does it? We just call various functions within master_background directly. So it's not obvious to me why this needs to be added to the cfg's and to the calwebb_spec2 pipeline module itself.

Reply:
Presuming that this should switchable, i.e., to perform or not perform, how would this be indicated? Would it be switched/piggy-backed on the existing `background` step?

Also, ideally, this should be encapsulated into a step. This wasn't done initially because it wasn't clear how exactly to implement, originally. This should be done.  However, since it is not done that way now, we can defer to whatever the answer is to the first question.

Reply:
Agree that it should be switchable, at least eventually, in case anyone ever does NOT want it applied. Also, is there existing logic in the flow that will skip all of the master bkg operations for MOS mode if the regular `background` step was applied previously (i.e. for simple nodding cases)? Separate from this PR, we also need to update the `calwebb_spec3` `master_background` step to test whether the master bkg has already been subtracted here in `calwebb_spec2` (for MOS mode) and skip the level-3 step. Finally, does the current `calwebb_spec2` set of updates include setting the S_xxxxxx keyword for `master_background` to COMPLETE if/when it gets applied? The level-3 step logic would need that to figure out if it's already been done.

Reply:
spec2: Will leave in the pseudo-mbs step. I will put in the logic to make both background steps mutually exclusive and add the S_XXXXX keyword when it is done.

spec3: Will file issue(s) to capture the necessary changes. Will these need be resolved for B7.6?

Reply:
I think at least the spec2 items should be resolved for B7.6. spec3 update should be easy, but isn't imperative for B7.6.